README.md

```diff
@@ -13,6 +13,7 @@
    npm install express 
    npm install discord.js
    npm install dotenv
+   npm install request
```

Comment:
This package has been deprecated it seems......

Replace with another package like `axios`